app/queries/decidim/action_delegator/delegates_votes.rb

```diff
@@ -4,9 +4,17 @@ module Decidim
   module ActionDelegator
     class DelegatesVotes < Rectify::Query
```

Comment:
We can replace this one with `DelegationVotes` but it may be a good idea to rename it instead so it doesn't get lost in the Git history.